package.json

```diff
@@ -153,7 +153,7 @@
     "type-lint": "tsc",
     "typings-test": "tsc -p tsconfig.types.test.json",
     "module-typings-test": "tsc -p tsconfig.module.test.json",
-    "spellcheck": "cspell \"**/*\"",
+    "spellcheck": "cspell --no-progress \"**\"",
```

Comment:
new verison prints all files to console, it is annoying, so with `--no-progress` we have only **file with problems**